MAUI/ListView/item-drag-and-drop.md

```diff
@@ -0,0 +1,485 @@
+---
+layout: post
+title: Item Reordering in .NET MAUI ListView control | Syncfusion
+description: Learn here all about Item Reordering support in Syncfusion .NET MAUI ListView (SfListView) control and more.
```

Comment:
description: Learn here all about Item Reordering support in the Syncfusion .NET MAUI ListView (SfListView) control and more.

Reply:
Addressed the concerns provided.